instrumentation/net/http/otelhttp/labeler.go

```diff
@@ -48,7 +48,10 @@ type labelerContextKeyType int
 
 const lablelerContextKey labelerContextKeyType = 0
 
-func injectLabeler(ctx context.Context, l *Labeler) context.Context {
+// InjectLabelerIntoContext helps to inject a Labeler into a context. Being a public
+// function, it is safe to use in any package that wants to add custom attributes to
+// the metrics recorded by the net/http instrumentation.
+func InjectLabelerIntoContext(ctx context.Context, l *Labeler) context.Context {
```

Comment:
How about naming this methid `ContextWithLabeler`?
That would better match the similar method names we have in the trace API.
https://github.com/open-telemetry/opentelemetry-go/blob/main/trace/context.go

Reply:
Yeah, you're right! I'll update the PR accordingly!